server/webapp/WEB-INF/rails.new/webpack/models/dashboard/personalization.js

```diff
@@ -57,6 +61,10 @@ function Personalization(filters, pipelineGroups) {
       filterSet = filters; // only changes local copy when successful
     });
   };
+
+  this.isPipelineVisible = (filterName, pipeline) => {
```

Comment:
This feels like it belongs on DashboardFilter, not Personalization as far as responsibilities go.

Reply:
The only issue I have with this is that Personalization is supposed to be the only interface to DashboardFilters and DashboardFilter. Removing this method from Personalization would mean the PersonalizationVM would be calling methods on the DashboardFilter.... Thoughts?

Reply:
My comment applies to the bellow comments too. Do we want to maintain Personalization as the only interface to interact with DashboardFilter(s), or are we okay with changing that